src/Illuminate/Support/Arr.php

```diff
@@ -657,4 +657,21 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+    
+    /**
+     * Loop through multiple arrays in parallel
+     *
+     * @param iterable ...$arrays
+     * @return array
+     */
+    public static function map(callable $callback, ...$arrays)
+    {
+        $result = [];
+
+        foreach (array_map(null, ...$arrays) as $values) {
+            array_push($result, call_user_func_array($callback, $values));
```

Comment:
Why are you using array_push here?

Reply:
`array_push` can be replaced with 
```
$result[] = call_user_func_array($callback, $values);
```
And I replaced it.